app/controllers/spree/orders_controller.rb

```diff
@@ -14,7 +14,7 @@ class OrdersController < ::BaseController
     respond_to :html
     respond_to :json
 
-    before_action :update_distribution, only: :update
+    before_action :set_current_order, only: :update
     before_action :filter_order_params, only: :update
```

Comment:
can you move these before_actions, only: update to beginning of the update method?